csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -421,6 +453,7 @@ public OrtLoggingLevel LogSeverityLevel
         /// Log Verbosity Level for the session logs. Default = 0. Valid values are >=0.
         /// This takes into effect only when the LogSeverityLevel is set to ORT_LOGGING_LEVEL_VERBOSE.
         /// </summary>
+        /// <value>returns _logVerbosityLevel value</value>
```

Comment:
nit: a lot of these <value> comments seem to not provide useful information, just implementation details